src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -403,42 +404,6 @@ def wait_for_create_job_to_complete(
     )
 
 
-def _hash_attachments(
```

Comment:
This function is used in a few CLIs moving forward, so to reduce confusion, I moved it to its own file.